website/content/docs/commands/init.mdx

```diff
@@ -1,99 +1,54 @@
 ---
 description: |
-  The `packer init` Packer command is used to download Packer plugin binaries.
-page_title: packer init - Commands
+    Reference information about the `packer init` command, which downloads and installs Packer one or more plugin binaries specified in a Packer template written in HCL.
+page_title: packer init command line reference
 ---
 
-# `init` Command
+# `init` command reference
 
--> **Note:** Packer init does not work with legacy JSON templates. You can
-upgrade your JSON template files to HCL using the [hcl2_upgrade](/packer/docs/commands/hcl2_upgrade) command.
+The `packer init` command initializes Packer according to an HCL template configuration. Refer to [Installing Plugins](/packer/docs/plugins/install) for additional information about installing plugins.
 
--> **Note:** Packer init will only work with multi-component plugins -- that is
-plugins that are named `packer-plugin-*`. To install a single-component plugin --
-that is `packer-provisioner-*`, `packer-builder-*`, etc. -- nothing changes, you will
-have to [install the plugin manually](/packer/docs/plugins#installing-plugins).
+## Description
 
-The `packer init` command is used to download Packer plugin binaries. This is
-the first command that should be executed when working with a new or existing
-template. This command is always safe to run multiple times. Though subsequent
-runs may give errors, this command will never delete anything.
+Use the `packer init` command to download and install plugins according to the `required_plugins` block in Packer templates written in HCL. Refer to [Specifying plugin requirements](/packer/docs/templates/hcl_templates/blocks/packer#specifying-plugin-requirements) in the template configuration reference for additional information about configuring the `required_plugins` block.
 
-You should invoke `packer init` on either an HCL2 template, or a directory that contains
-at least a valid HCL2 template, and eventually other related dependencies like varfiles
-for example.
+Legacy JSON templates are not supported. You can convert your JSON template files to HCL using the [hcl2_upgrade](/packer/docs/commands/hcl2_upgrade) command.
 
-Example:
+We recommend running the `packer init` command as the first step when working with a new or existing template. You can run the command multiple times. Subsequent runs may produce errors, but the command never deletes already-installed plugins.
 
-```sh
-$ ls .
-template.pkr.hcl varfile.pkrvars.pkr.hcl
+### Third-party plugin verification
 
-$ packer init template.pkr.hcl # You can invoke packer init on a single template in this case
-                               # This works if the template is self-contained, but may fail if
-                               # the template is meant to be built as a bundle of partials.
+We recommend that you vet and verify any third-party plugins you want to install.
```

Comment:
For context: I had left a comment on a previous version, I'm not sure we should include the "verifies" phrasing as technically there's not anything we verify, and Packer happily loads anything that quacks like a plugin, so we ended up removing this part, only to focus on the warning about third-party plugins since we don't vet them.

Reply:
So I think we might be conflating the term verifies here. Looking at the sentence "We recommend that you vet and verify any third-party plugins you want to install." I take that to mean that user should validate that the external plugin does what it says and that it is free of any malicious code.  Which is wording we use to have on our plugins docs when we had the tiers overview. 

But it doesn't look like that is the case anymore. That said, that is what I take "HashiCorp only verifies plugins from GitHub under the`hashicorp/*` namespace" to mean. We verify that any plugin under the HashiCorp org on GitHub can be used by Packer, it is free of malicious code, and does exactly what it is intended to do. 

Is that not what we are trying to say here? 


If we are focusing solely on plugin installation validation we should inform the user that Packer only checks that the external plugins contain a valid SHA256SUM file of the specified version constraint. We recommend that you verify external plugins satisfy your expected use case.

Reply:
I'm afraid `verify` will be understood possibly as "check signatures" or something like that, which we don't do at all on plugins, as plugin binaries aren't signed at all IIRC, only Packer core is.
That being said I understand your point of view, and it makes sense, but if we're to avoid any kind of misunderstanding here, I would focus on community/third-party plugins as inherently sketchier because that's a normal repo on Github maintained by volunteers.
Maybe just saying `vet your binaries` would be enough, it's more of a disclaimer than anything.